heartbeat/exportfs

```diff
@@ -311,6 +344,28 @@ export_one() {
 	ocf_log info "directory $dir exported"
 	return $OCF_SUCCESS
 }
+
+merge_notify_list()
+{
+	local dir=$1
+	local subdir=${OCF_RESKEY_state_subdir}
+
+	if [ ! -d $dir/$subdir/statd ]; then
+		mkdir -p $dir/$subdir/statd/sm
+		chown -R rpcuser.rpcuser $dir/$subdir/statd
+	fi
+	if selinuxenabled; then
```

Comment:
This should be changed to avoid running selinuxenabled on non-systemd systems, e.g.:
https://github.com/ClusterLabs/resource-agents/blob/805bfa13543d2356eb4b7e72c040f829a24a3c69/heartbeat/nfsserver#L198-L202